android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -262,12 +288,31 @@ public NetworkModel getNetworkModel() {
         return mNetworkModel;
     }
 
+    public SendModel creteSendModel() {
```

Comment:
`createSendModel`?

Reply:
to create and get a sendModel, I want a send model to have mode things specific to a particular transaction, it will have the token or asset for a transaction.

Reply:
what @wchen342 meant is that you have a typo, `crete` instead of `create`

Reply:
lol, my bad, thanks!